test/e2e/storage/testsuites/subpath.go

```diff
@@ -903,6 +908,9 @@ func testSubpathReconstruction(f *framework.Framework, pod *v1.Pod, forceDelete
 	framework.ExpectNoError(err, "while getting pod")
 
 	utils.TestVolumeUnmountsFromDeletedPodWithForceOption(f.ClientSet, f, pod, forceDelete, true)
+
+	mountPointsAfter := utils.FindVolumeGlobalMountPoints(f.ClientSet, pod)
+	gomega.Expect(mountPointsAfter).To(gomega.ConsistOf(mountPoints), "Global mount points leaked. Before: %v, After: %v.", mountPoints, mountPointsAfter)
```

Comment:
I wonder if it would be easier to use `sets.String` and expect their difference is zero. Global directory can be quite big and noticing an extra directory could be hard.

Reply:
I think it's a good idea, easier to debug flaky tests.